test/dotnet/scenarios.json

```diff
@@ -81,5 +81,15 @@
                 "aspnetcoreRuntimeVersions": "latest"
             }
         }
+    },
+    "install_dotnet_workloads": {
```

Comment:
Looks like the tests are failing, can you help fix them?

Reply:
Thanks for the hint